src/Components/Samples/BlazorUnitedApp/Pages/Error.razor

```diff
@@ -0,0 +1,35 @@
+﻿@page "/Error"
```

Comment:
One case I think won't work is `<Router @rendermode="@(new WhateverRenderMode(prerender: false))" />`.

Without prerendering, I think what will happen is:

 * Error middleware will change the endpoint to `/Error`
 * This will match against RazorComponentEndpoint, which will render `App`, which will try to render the `Router` without prerendering, so the user will just see a blank screen. I think they'll have a 500 status code though, right?

Do you agree this is what will happen? I'm not sure what we can do about it, since even if we detect "this is an error page but you were trying to render an interactive, non-prerendered component" it's not certain we'd want to throw a different error. Maybe the layout contains an interactive, non-prerendered component for unrelated reasons, and we'd be better off just ignoring it.

Reply:
Note that we can't force prerendering to be on, either, since the layout might include some WebAssembly components that don't support being prerendered on the server.

Reply:
I'd be fine with concluding that people who want a non-prerendered router can't use `.razor` for their error pages. We can document this, though admittedly that doesn't do much for discoverability.

Reply:
Yes I think so, but I think we should just document that.